christfellowshipchurch/web-app-v2
# Locations Metadata

### About
This PR makes it so the metadata for the locations pages comes primarily from Rock, if the fetch fails or there is no metadata in Rock for that location, the old hardcoded metadata will be used instead.

### Test Instructions
1. Make a change for any location page in the [SEO](https://rock.christfellowship.church/page/1780) page in Rock, easiest to test is the title, just look at the tab in Chrome and the same should change.
2. Test that nothing changed in that page in the live page.
3. Test that the change took effect in the testing page.

### Closes Tickets
[CFDP-2985](https://christfellowshipchurch.atlassian.net/browse/CFDP-2985)


[CFDP-2985]: https://christfellowshipchurch.atlassian.net/browse/CFDP-2985?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
